featuremanagement/_models/_targeting_context.py

```diff
@@ -14,6 +14,17 @@ class TargetingContext(NamedTuple):
 
     # The user ID
     user_id: str = ""
+    """
```

Comment:
Not sure but should this doc string rather be on top? Also, I see we already have the same description above on line 15. Should we just move this docstring there,

Reply:
This doc string needs to be here for sphinx to pick it up. I already tested it.

Reply:
https://microsoft.github.io/FeatureManagement-Python/html/index.html#featuremanagement.TargetingContext

Reply:
Then perhaps, we can remove the comment on top if that is the case.

Reply:
Refresh the PR, I already pushed the change.